### PR TITLE
Remove ML_DEBUG_PRINT from pxt.json & improves debug msgs.

### PR DESCRIPTION
**Pull request by microbit-carlos**
Removing `ML_DEBUG_PRINT` config from pxt.json let's user programmes add it later in their pxt.json file within the MakeCode editor (user cannot edit extension files in MakeCode).

A `[ML] ` prefix is added to all serial debug lines, so that they are easier to distinguish from user generated serial data.

Also adds debug info indicating when the accelerometer sampling rate has drifted (lower or higher than expected).

As debug messages are disabled by default, this shouldn't have any effect on "normal" user programmes.

### CODE REVIEW

**Comment by microbit-robert** [comment]
@microbit-carlos Is there an easy way to test this?

**Comment by microbit-carlos** [comment]
Building the extension locally with the `"ML_DEBUG_PRINT": 1` entry added to pxt.json.
The current example model and autogenerated code in this repo does still work, but we might want to update it with the latest changes at some point.

**Comment by microbit-robert** [comment]
> Building the extension locally with the `"ML_DEBUG_PRINT": 1` entry added to pxt.json. The current example model and autogenerated code in this repo does still work, but we might want to update it with the latest changes at some point.

I think this is just changing the pxt.json for the extension, not the 'users' pxt.json.

Please can you try with this branch of CreateAI which targets this branch of the extension? https://review-createai.microbit.org/debug-updates-test

**Comment by microbit-carlos** [comment]
Oh, right, I thought you meant how to test the new debug messages.

Yes, to test it with a user programme we need to add the same flag to their pxt.json (the project settings in text view)

image

```json
    "yotta": {
        "config": {
            "ML_DEBUG_PRINT": 1
        }
    }
```

**Comment by microbit-carlos** [comment]
The problem might be getting MakeCode to rebuild CODAL in the cloud, I usually add the https://github.com/carlosperate/pxt-banana/ extension and that might trigger it.

File: pxtextension.cpp [hunk 1]
```diff
@@ -125,6 +138,7 @@ namespace mlrunner {
         if (!(++samplesTaken % mlSampleCountsPerInference) && mlDataProcessor.isDataReady()) {
             runModel();
         }
+        DEBUG_PRINT("S: %d ms\n", uBit.systemTime() - lastSampleTime);
```
Comment (microbit-carlos) [inline pxtextension.cpp]: I'll remove this one, is not really needed anymore

Comment (microbit-carlos) [inline pxtextension.cpp]:
```suggestion
        }
```

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**